TeamProject/GlobalWarmingGame/BuildingManager.cs

```diff
@@ -0,0 +1,95 @@
+﻿using Engine.TileGrid;
+using GlobalWarmingGame.Action;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GlobalWarmingGame.Interactions.Interactables.Buildings
```

Comment:
Namespace doesn't match folder location. This class should be moved to TeamProject/GlobalWarmingGame/Interactions/Interactables/Buildings/